runtime/bcutil/defineclass.c

```diff
@@ -772,7 +772,9 @@ callDynamicLoader(J9VMThread *vmThread, J9LoadROMClassData *loadData, U_8 * inte
 			}
 		}
 
-		if (J9_ARE_ANY_BITS_SET(vm->extendedRuntimeFlags, J9_EXTENDED_RUNTIME_RECREATE_CLASSFILE_ONLOAD)) {
+		if (J9_ARE_ANY_BITS_SET(vm->extendedRuntimeFlags, J9_EXTENDED_RUNTIME_RECREATE_CLASSFILE_ONLOAD)
+			&& ((NULL == localBuffer) || (NULL == localBuffer->patchMap))
```

Comment:
Looking at the code,  I don't see any cases where `localBuffer` could be NULL, so the NULL check may be redundant. Though I'm not against adding a NULL check.
Also can you add a comment here to explain the reason for skipping this operation.

Reply:
I think I feel better about keeping the NULL check on `localBuffer`. But I will add a comment there.